src/Util.ts

```diff
@@ -681,9 +563,39 @@ export function launchWebUrl(url) {
 }
 
 export function launchMusicAnalytics() {
+  const isRegistered = checkRegistration();
+  if (!isRegistered) {
+    return;
+  }
   open(`${launch_url}/music`);
 }
 
+export function checkRegistration(showSignup = true) {
+  if (!getItem("name")) {
+    if (showSignup) {
+      showModalSignupPrompt("A registered account is required to continue. Sign up or log in to continue.");
```

Comment:
Sign up or register for a web.com account at Software.com to view your most productive music.

Reply:
ok, cool. will update it to that

Reply:
updated, thanks